http-client/src/main/java/io/micronaut/http/client/exceptions/HttpClientResponseException.java

```diff
@@ -31,23 +29,24 @@
  */
 public class HttpClientResponseException extends HttpClientException implements HttpResponseProvider {
```

Comment:
You should probably add a `getRequest()` method?

Reply:
Done.